android/autodispose-android-archcomponents/src/main/java/com/uber/autodispose/android/lifecycle/LifecycleEventsObservable.java

```diff
@@ -34,7 +34,7 @@
 import io.reactivex.Observer;
 import io.reactivex.subjects.BehaviorSubject;
 
-@RestrictTo(LIBRARY)
+@RestrictTo(LIBRARY_GROUP)
```

Comment:
This throws a lint error now. `LIBRARY_GROUP` seems correct since we're still restricting it to `com.uber.autodispose` gradle id.